src/Microsoft.AspNetCore.Razor.Language/Legacy/EditResult.cs

```diff
@@ -1,17 +1,26 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNetCore.Razor.Language.Syntax;
+
 namespace Microsoft.AspNetCore.Razor.Language.Legacy
 {
     internal class EditResult
     {
+        public EditResult(PartialParseResultInternal result, SyntaxNode editedNode)
+        {
+            Result = result;
+            EditedNode = editedNode;
+        }
+
         public EditResult(PartialParseResultInternal result, SpanBuilder editedSpan)
```

Comment:
I missed removing this and there may be few other unused legacy methods. I'll remove those as part of cleanup.